payment/src/main/java/net/optile/payment/ui/widget/InputLayoutWidget.java

```diff
@@ -125,7 +173,8 @@ void setTextInputMode(EditTextInputMode mode) {
 
     String getValue() {
         CharSequence cs = textInput.getText();
-        return cs != null ? cs.toString().trim() : "";
+        String val = cs != null ? cs.toString().trim() : "";
+        return mode != null ? mode.normalize(val) : val;
```

Comment:
Brackets would make this a bit more readable:
```suggestion
        String val = (cs != null) ? cs.toString().trim() : "";
        return (mode != null) ? mode.normalize(val) : val;
```

Reply:
Fixed